javascript/slides/varsfunctions.html

```diff
@@ -196,7 +196,34 @@
                                                             </section>
                                                         </div>
 
-
+                                                        <div class="slide normal"> 
+                                                        <header><h1>Strings</h1></header> 
+                                                         <section class="content"> 
+                                                           <p>A string holds an ordered list of character:
```

Comment:
I think calling a string an array of characters leaves the possibility of confusion when we later introduce the array. I'm at a loss for another synonym for ordered.